TelCo.ColorCoder/EvenCountColorTest.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using System.Diagnostics;
+using System.Drawing;
+
+namespace TelCo.ColorCoder
+{
+    class EvenCountColortest
+    {
+        private static void Main(string[] args)
+        {
+            ColorManual.printManual();
```

Comment:
Try out an assert for this

Reply:
I have few thoughts about this
1. I can store whatever the printManual() is printing in a data structure or string and manually write and assert all 25 pairs in the main testing program.
2. I can create a new method testManual() in this class that loops in the same way printManual() does but test each iteration by using the getpairnumber() method.
3. Similar to point 2, instead of using getpairnumber(), I use getcolorpair() and test each iteration.

If I implement point 2 or 3, then the testing isn't happening in the main testing program but in the ColorManual program, Isn't that wrong? Since I won't be using assert in main testing program. 
If I implement point 1, it will lot of hardcoding. 
Do you have any suggestions?